package.json

```diff
@@ -21,7 +21,7 @@
     }
   },
   "scripts": {
-    "test": "npm run test:flow && npm run test:unit && npm run test:ts",
```

Comment:
Temp disabled flow

Reply:
Happy to move to TS. Is on my todo list anyways